website/templates/project/registration_editor_templates.mako

```diff
@@ -144,4 +147,21 @@
     </div>
 </script>
 
+<script type="text/html" id="deleted">
+	<% from datetime import datetime %>
```

Comment:
You may be confusing mako-style templating with KO templates. 

the ```<% cmd %>``` syntax is a mako command and is computed (once and only once) server side. In this case all deleted comments will have the same deleted date.